lib/backend/syncersv1/updateprocessors/globalnetworkpolicyprocessor.go

```diff
@@ -43,16 +44,19 @@ func convertGlobalNetworkPolicyV2ToV1Value(val interface{}) (interface{}, error)
 	if !ok {
 		return nil, errors.New("Value is not a valid GlobalNetworkPolicy resource value")
 	}
-	return convertGlobalPolicyV2ToV1Spec(v3res.Spec)
-}
 
-func convertGlobalPolicyV2ToV1Spec(spec apiv3.GlobalNetworkPolicySpec) (*model.Policy, error) {
+	spec := v3res.Spec
+	selector := spec.Selector
+
+	selector = PrefixAndAppendSelector(selector, spec.NamespaceSelector, conversion.NamespaceLabelPrefix)
```

Comment:
I think we need to do something special here for the namespace selector, based on my reading of this comment: https://github.com/projectcalico/libcalico-go/blob/d96206686f6f1769b2760630d8be9212ce94d3ff/lib/backend/syncersv1/updateprocessors/rules.go#L54-L57

Reply:
Fixed, I think